ballerina/utils.bal

```diff
@@ -0,0 +1,217 @@
+// Copyright (c) 2023, WSO2 LLC. (http://www.wso2.com) All Rights Reserved.
+//
+// WSO2 LLC. licenses this file to you under the Apache License,
+// Version 2.0 (the "License"); you may not use this file except
+// in compliance with the License.
+// You may obtain a copy of the License at
+//
+// http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing,
+// software distributed under the License is distributed on an
+// "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY
+// KIND, either express or implied.  See the License for the
+// specific language governing permissions and limitations
+// under the License.
+
+import ballerina/url;
+
+type SimpleBasicType string|boolean|int|float|decimal;
+
+# Represents encoding mechanism details.
+type Encoding record {
+    # Defines how multiple values are delimited
+    string style = FORM;
```

Comment:
Shouldn't this be `EncodingStyle`?
```suggestion
    EncodingStyle style = FORM;
```

Reply:
Fixed in [2adfefa](https://github.com/ballerina-platform/module-ballerinax-docusign.dsclick/pull/1/commits/2adfefa9bd1f9d8fd566dbf634d363d21553a08a)